src/main/java/com/ibm/usecases/compliance/service/CompliancePreparationService.java

```diff
@@ -42,9 +42,9 @@ public final class CompliancePreparationService {
 
     public Collection<CryptographicAsset> receiveCryptographicAssets(
             @Nonnull ICBOMReadRepository readRepository,
-            @Nonnull GitUrl gitUrl,
+            @Nonnull ScanUrl gitUrl,
```

Comment:
Same as "For compliance either a giturl or purl should be the required input. A scanUrl should not be existing in the "compliance" domain"

Reply:
I reactivated the old GitUrl class to represent the gitUrl in ScanAggregate which is passed in here.